src/Chapter1.hs

```diff
@@ -477,8 +479,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod n 10
```

Comment:
Nice! 💪🏼 
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. But it is easily fixed with one standard function, that gives the absolute values 😉  Sometimes corner cases can be tricky to spot and fix...